packages/react-components/src/components/chart/trendCursor/resize/useHandleResize.ts

```diff
@@ -7,36 +7,33 @@ import { calculateSeriesMakers } from '../calculations/calculateSeriesMakers';
 
 const useHandleResize = ({ size, series, graphic, setGraphic, chartRef, visualization }: TrendCursorProps) => {
   const prevSize = useRef(size);
-  // to avoid unnecessary re-rendering
-  if (size.width !== prevSize.current.width || size.height !== prevSize.current.height) {
-    const newG = graphic.map((g) => {
-      // splitting into two separate if block to avoid calculations when not necessary
 
-      // if height has changed, update Y values
-      if (size.height !== prevSize.current.height) {
+  // delaying the updating by 10ms, so that the graph is drawn before reading the values back
```

Comment:
What guarantee do you have that the graph will be drawn first? Are we using the setTimeout to control the event loop?

Reply:
no guarantee, i just need one render cycle... 10ms is an arbitrary time to let one render cycle pass...

Reply:
Seems like an area which may result in bugs in the future. You might be able to get around specifying a time to wait by using https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame.

Reply:
but requestAnimationFrame lets me know how many times something got render or will render... I cannot skip a frame ? hmmm, let me try.. If there is a alternate solution to using settimeout il take it